src/utils/lesson-metadata.ts

```diff
@@ -0,0 +1,65 @@
+import {LessonResource} from 'types'
+import some from 'lodash/some'
+
+export const mergeLessonMetadata = (
```

Comment:
Move `mergeLessonMetadata` into its own module. That makes it easier to test and declutters the `lib/lesson.ts` module.